### PR TITLE
1.11.7 release

### PR DESCRIPTION
**Pull request by NSeydoux**
This PR bumps the version to 1.11.7.

# Checklist

- [X] I used `npm run lerna-version -- <major|minor|patch> --no-push` to update the version number, first inspecting the CHANGELOG to determine if the release was major, minor or patch.
- [X] The CHANGELOG has been updated to show version and release date - https://keepachangelog.com/en/1.0.0/.
- [X] The **only** commits in this PR are:
  - the CHANGELOG update.
  - the version update.
- [ ] Once this PR is merged, I will push the tag created by `lerna version ...` (e.g. `git push origin vX.X.X`).

### CODE REVIEW

File: .github/workflows/e2e-browser.yml [hunk 1]
```diff
@@ -32,9 +32,7 @@ jobs:
       - run: npm run bootstrap -- --ci
       - run: npm run build
       - name: Install e2e tests dependencies
-        run: |
-          npm ci
-          npx playwright install --with-deps
+        run: npx playwright install --with-deps
```
Comment (matthieubosquet) [inline .github/workflows/e2e-browser.yml:35]: I guess this should have been in another PR.

**Review by matthieubosquet — APPROVED** [review]
Wondering if it's ok to have your ci change as part of this PR?

**Review by ThisIsMissEm — APPROVED** [review]
LGTM

**Comment by ThisIsMissEm** [comment]
@NSeydoux you may wish to kick the CI off again here though, as something happened in the verify-imports step.

**Pull request merged**